README.md

```diff
@@ -24,3 +24,504 @@ You can also learn more about our [Governance](/GOVERNANCE.md) structure.
+
+##### 2.1.1 Create an OpenID Connect Provider for Your Cluster
+
+```sh
+eksctl utils associate-iam-oidc-provider --cluster ${CLUSTER_NAME} \
+--region ${AWS_DEFAULT_REGION} --approve
+```
+
+Get the OIDC ID
+```sh
+AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+OIDC_PROVIDER=$(aws eks describe-cluster --name $CLUSTER_NAME --region $AWS_DEFAULT_REGION \
+--query "cluster.identity.oidc.issuer" --output text | cut -c9-)
+```
+
+##### 2.1.2 Create an IAM Role
+
+Create a file named trust.json and insert the following trust relationship code block required for IAM role into it.
+```sh
+printf '{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Federated": "arn:aws:iam::'$AWS_ACCOUNT_ID':oidc-provider/'$OIDC_PROVIDER'"
+      },
+      "Action": "sts:AssumeRoleWithWebIdentity",
+      "Condition": {
+        "StringEquals": {
+          "'$OIDC_PROVIDER':aud": "sts.amazonaws.com",
+          "'$OIDC_PROVIDER':sub": [
+            "system:serviceaccount:ack-system:ack-sagemaker-controller",
+            "system:serviceaccount:ack-system:ack-applicationautoscaling-controller"
+          ]
+        }
+      }
+    }
+  ]
+}
+' > ./trust.json
+```
+
+
+Run the following command to create a role with the trust relationship defined in `trust.json`. This role enables the Amazon EKS cluster to get and refresh credentials from IAM.
+```sh
+OIDC_ROLE_NAME=ack-sage-role-$CLUSTER_NAME
+
+aws --region $AWS_DEFAULT_REGION iam create-role --role-name $OIDC_ROLE_NAME --assume-role-policy-document file://trust.json
+
+# Attach the AmazonSageMakerFullAccess Policy to the Role
+aws --region $AWS_DEFAULT_REGION iam attach-role-policy --role-name $OIDC_ROLE_NAME --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
+export OIDC_ROLE_ARN=$(aws --region $AWS_DEFAULT_REGION iam get-role --role-name $OIDC_ROLE_NAME --output text --query 'Role.Arn')
+echo $OIDC_ROLE_ARN
+```
+
+Take note of OIDC_ROLE_ARN printed in the previous step; you will pass this value to the service account used by the controller.
+
+### 3.0 Install Controllers
+
+#### 3.1 Install SageMaker Controller
+
+##### 3.1.1 Download helm chart
+
+```sh
+export HELM_EXPERIMENTAL_OCI=1
+export SERVICE=sagemaker
+export RELEASE_VERSION=v0.0.1
+export CHART_EXPORT_PATH=/tmp/chart
+export CHART_REPO=public.ecr.aws/aws-controllers-k8s/chart
+export CHART_REF=$CHART_REPO:$SERVICE-$RELEASE_VERSION
+
+mkdir -p $CHART_EXPORT_PATH
+helm chart pull $CHART_REF
+helm chart list
+helm chart export $CHART_REF --destination $CHART_EXPORT_PATH
+```
+
+##### 3.1.2 Choose one of the two options for deployment
+
+  - [Option 1] Cluster scoped deployment
+    - ```sh
+      # Change values in helm chart
+      vim $CHART_EXPORT_PATH/ack-$SERVICE-controller/values.yaml
+
+      # line #29, #30, change the aws.region, aws.account_id
```

Comment:
Why not just override these using `--set` when doing `helm install`? That way everything can stay in environment variables

Reply:
`serviceaccount` has  `{}` which needs to be removed. It didn't work for me